src/master/chartsdata.cc

```diff
@@ -18,6 +18,9 @@
    along with SaunaFS  If not, see <http://www.gnu.org/licenses/>.
  */
 
+
+/* DEPRECATED: Use src/metrics instead */
```

Comment:
I think this comment is somehow ambiguous. 
If the comment is for file `master/charsdata.h`, the comment should be placed beside the file which is being deprecated. Otherwise, it should be explicitely mentioned the name of file(s) that are being deprecated.